setup.py

```diff
@@ -14,7 +14,7 @@
 if sys.version_info[:2] == (2, 7) or sys.version_info[:2] <= (3, 5):
     INSTALL_REQUIRES=['numpy>=1.10,<1.24', 'biopython<=1.76', 'pyparsing', 'scipy']
 else:
-    INSTALL_REQUIRES=['numpy>=1.10,<1.24', 'biopython<=1.79', 'pyparsing', 'scipy', 'setuptools']
+    INSTALL_REQUIRES=['numpy>=1.10,<1.24', 'biopython<=1.79', 'pyparsing', 'scipy', 'setuptools', 'mmtf-python']
```

Comment:
You probably don’t need to add another hard dependency as you’re using try/except to import it

Reply:
So where should this dependency be put so that the tests can pass? In `.github/workflows/main.yml`?